src/ontology/mondo-edit.obo

```diff
@@ -520696,6 +520702,7 @@ intersection_of: MONDO:0020573 ! inherited disease susceptibility
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/16882 ! HCN4
 intersection_of: predisposes_towards MONDO:0005579 ! idiopathic generalized epilepsy
 relationship: excluded_subClassOf MONDO:0005579 {source="OMIM:619521", source="https://orcid.org/0000-0001-5208-3432"} ! idiopathic generalized epilepsy
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/16882 ! HCN4
```

Comment:
mondo.owl v2024-12-03 does not have any source provenance for this gene association